bin/ck8s

```diff
@@ -13,9 +13,9 @@ usage() {
     echo "  init                                            initialize the config path" 1>&2
     echo "  bootstrap <wc|sc>                               bootstrap the cluster" 1>&2
     echo "  apps <wc|sc> [--skip-template-validate]         deploy the applications" 1>&2
-    echo "  apply <wc|sc> [--skip-template-validate]        bootstrap and apps" 1>&2
+    echo "  apply <wc|sc> [--sync] [--skip-template-validate] bootstrap and apps" 1>&2
     echo "  test <wc|sc>                                    test the applications" 1>&2
```

Comment:
```suggestion
    echo "  apply <wc|sc> [--sync] [--skip-template-validate] bootstrap and apps" 1>&2
    echo "  test <wc|sc>                                      test the applications" 1>&2
```
Nit: fix the indentation for the whole `usage` text (I can't mark all lines in this comment) so that it matches `bootstrap and apps`